tests/rust/tcp-test/close/mod.rs

```diff
@@ -171,3 +179,167 @@ fn close_connecting_socket(libos: &mut LibOS, remote: &SocketAddrV4) -> Result<(
 
     Ok(())
 }
+
+/// Attempts to close an invalid queue descriptor.
+fn async_close_invalid_queue_descriptor(libos: &mut LibOS) -> Result<()> {
```

Comment:
In our code base, we use the following naming scheme:

- `qr` => Demikernel queue results
- `qt` => Demikernel queue tokens
- `qd` => Demikernel queue descriptors

In your code your are using `qresult` for Demikernel queue results. Could you change?